### PR TITLE
More documentation updates

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues

- https://github.com/cityofaustin/atd-data-tech/issues/19103
- https://github.com/cityofaustin/atd-data-tech/issues/19106

## Testing

**URL to test:** 

**Steps to test:**

---

#### Ship list

- [ ] Check migrations for any conflicts with latest migrations in `main` branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: database/README.md [hunk 1]
```diff
@@ -199,13 +233,17 @@ The view `crashes_change_log_view` provides a unioned view of the unified table
...
+| `council_districts`   | polgyon       | City of Austin council districts                                                                          | ArcGIS Online authoritative layer owned by CTM GIS                   |
+| `atd_jurisdictions`   | polgyon       | City of Austin jurisdictions                                                                              | ArcGIS Online authoritative layer owned by CTM GIS                   |
+| `engineering_areas`   | polgyon       | TPW traffic engineering areas                                                                             | ArcGIS Online authoritative layer owned by DTS GIS                   |
+| `non_coa_roadways`    | polgyon       | Polygon layer covering roadways which are not maintained by the City of Austin                            | ArcGIS Online authoritative layer maintained by Vision Zero GIS team |
+| `atd_txdot_locations` | polgyon       | Aka, "location polygons", these shapes are used to group crashes based on an intersection or road segment | ArcGIS Online authoritative layer maintained by Vision Zero GIS team |
```
Comment (johnclary) [inline database/README.md:246]: i considered going into depth about the trigger functions, but i think it's probably best for a dev to inspect these tables to know for sure 🤷

File: database/README.md [hunk 2]
```diff
@@ -75,7 +75,41 @@ The process for updating `units` and `people` behaves in the same manner as `cra
...
+  - To set up a recurring request, add a range of dates that ends in the future.
+
+Any part of the range that falls in the past will be delivered in single zip that is separate from the zips that will deliver in the future. The includes - all records with process dates available including today.
+
+Any part of the range that is in the future will create daily zips that include each day available going forward. For example, on 4/19/2024, you make a request for Process Begin Date = 01/01/2024 and Process End Date = 12/31/2024 The would receive two zips: One containing all records with process date from 01/01/2024 to 04/18/2024, and one containing all records with process date from 04/19/2024 to 04/19/2024. Going forward, you will receive one zip per day for each process date that passes
+
+- **Extract password**: the password called `EXTRACT_PASSWORD` from Vision Zero CRIS Import 1Password item
+- **Delivery**: How you want to receive it. Typically you would use the pre-configured AWS option, specifiyng the `dev`, `staging`, or `prod` inbox subdirectory. See the CRIS import ETL readme for more details.
```
Comment (johnclary) [inline database/README.md:112]: This all used to be true, and will again be true assuming we magically get a resolution to https://github.com/cityofaustin/atd-data-tech/issues/19150

Comment (johnclary) [reply database/README.md:112]: also h/t to whoever last wrote this up in gitbook, where it was largely copied from.

File: database/README.md [hunk 3]
```diff
@@ -75,7 +75,41 @@ The process for updating `units` and `people` behaves in the same manner as `cra
 
 #### CRIS Extract configuration and accounts
 
-TODO. See https://app.gitbook.com/o/-LzDQOVGhTudbKRDGpUA/s/-M4Ul-hSBiM-3KkOynqS/vision-zero-crash-database/visionzero-sftp-etl#getting-extracts-manually
+The team maintains multiple CRIS login accounts to manage the deliver of CRIS data to our AWS S3 ingest bucket. See the [CRIS import ETL documentation](../etl/cris_import/README.md) for more information on how CRIS data is delivered to S3 and imported into our database.
```
Comment (chiaberry) [inline database/README.md:78]: *delivery

File: database/README.md [hunk 4]
```diff
@@ -75,7 +75,41 @@ The process for updating `units` and `people` behaves in the same manner as `cra
...
+
+The credentials for our CRIS logins are in the password store, including a note in the title indicating each login's purpose:
+
+- **Production extract account**: this is the account which is configured for daily delivery of extracts to S3 production.
+- **Dev/testing extract account**: this accound should be used for requesting ad-hoc CRIS extracts for delivery via S3 or manual download.
```
Comment (chiaberry) [inline database/README.md:83]: *account

File: database/README.md [hunk 5]
```diff
@@ -75,7 +75,41 @@ The process for updating `units` and `people` behaves in the same manner as `cra
...
+- **Query & analyze account**: this account can be used for the CRIS query interface, that enables querying and access to individual crash records.
+
+Additional information about CRIS access can be found on the [TxDOT website](https://www.txdot.gov/data-maps/crash-reports-records/crash-data-analysis-statistics.html).
+
+Follow these steps too configure a new extract delivery:
```
Comment (chiaberry) [inline database/README.md:88]: *to

File: database/README.md (hunk 1 above)
Comment (chiaberry) [reply database/README.md:246]: Geometry type column has a minor typo repeated, polygon

**Comment by johnclary** [comment]
i fixed those horrible typos, removed a few `$` signs, and i fixed the `./vison-zero` script invocation that Mateo stumbled on recently.

File: README.md [hunk 6]
```diff
@@ -53,7 +53,7 @@ Note: the `-f / --filename` flag can be optionally used to point to a specific d
 4. Start the Vision Zero Editor
 
 ```shell
-$ vision-zero vze-up
+vision-zero vze-up
```
Comment (chiaberry) [inline README.md:56]: nice catch, this one needs the `./` treatment too

**Review by chiaberry — APPROVED** [review]
🚢 ✏️ 🚢

**Pull request merged**